test/ussd_health_worker.test.js

```diff
@@ -136,34 +136,11 @@ describe("familyconnect health worker app", function() {
                     })
                     .run();
             });
-            it("to state_msg_receiver", function() {
-                return tester
-                    .setup.user.addr('0820000111')
-                    .inputs(
-                        {session_event: 'new'}  // dial in
-                    )
-                    .check.interaction({
-                        state: 'state_msg_receiver',
-                        reply: [
-                            "Please select who will receive the messages on their phone:",
-                            "1. Head of the Household",
-                            "2. Mother to be",
-                            "3. Family member",
-                            "4. Trusted friend"
-                        ].join('\n')
-                    })
-                    .check(function(api) {
-                        go.utils.check_fixtures_used(api, [0]);
-                    })
-                    .run();
-            });
             it("to state_msisdn", function() {
                 return tester
                     .setup.user.addr('0820000111')
                     .inputs(
                         {session_event: 'new'}  // dial in
-                        , '12345'  // state_auth_code - personnel code
-                        , '1'  // state_msg_receiver - head of household
```

Comment:
I think this was actually an error in the test. The number above is a recognised number, so it shouldn't ask for a personnel code. I think what was happening is that it was asking for the message receiver, `12345` was put in as the input, which was not a valid input, so the head of household error screen was presented, where they put in `1`, which is a valid input, and so it went onto the next state as normal.